wolfcrypt/src/asn.c

```diff
@@ -6148,7 +6238,12 @@ static int HashForSignature(const byte* buf, word32 bufSz, word32 sigOID,
 static int ConfirmSignature(SignatureCtx* sigCtx,
     const byte* buf, word32 bufSz,
     const byte* key, word32 keySz, word32 keyOID,
-    const byte* sig, word32 sigSz, word32 sigOID)
+    const byte* sig, word32 sigSz, word32 sigOID
+#ifdef WOLFSSL_RENESAS_TSIP_TLS
+    ,
+    byte* tsip_encRsaKeyIdx
```

Comment:
I think it's a cleaner solution to make this argument a permanent addition to the ConfirmSignature() prototype, ie:

```
static int ConfirmSignature(SignatureCtx* sigCtx,
    ...
    byte* rsaKeyIdx);
```

Then, inside the ConfirmSignature() function, use a "(void)rsaKeyIdx" in the case where TSIP is not defined, ie:

```
#ifndef WOLFSSL_RENESAS_TSIP_TLS
    (void)rsaKeyIdx;
#endif
```

This makes all calling locations of ConfirmSignature a little cleaner by removing an #ifdef for TSIP.

Reply:
Agree. Removed #ifdef directive, and the argument is added permanently.